pilot/pkg/config/clusterregistry/conversion.go

```diff
@@ -84,6 +84,47 @@ func (cs *ClusterStore) GetPilotClusters() []*k8s_cr.Cluster {
 	return cs.clusters
 }
 
+// ReadClustersV2 reads multiple clusters based upon the label istio/multiCluster
+func ReadClustersV2(k8s kubernetes.Interface, cs *ClusterStore) (errList error) {
+	err := getClustersConfigsV2(k8s, cs)
+	if err != nil {
+		// Errors were encountered, but cluster store was populated
+		log.Errorf("The following errors were encountered during multicluster label processing: [ %v ]",
+			err)
+	}
+
+	return nil
+}
+
+// getClustersConfigsV2 reads mutiple clusters from secrets with labels
+func getClustersConfigsV2(k8s kubernetes.Interface, cs *ClusterStore) (errList error) {
+	listOptions := metav1.ListOptions{LabelSelector: "istio/multiCluster"}
```

Comment:
do not use string here, define `const`with value of label and use that const instead.